ui/webui/src/app.jsx

```diff
@@ -23,24 +23,39 @@ import {
 
 import { InstallationLanguage } from "./InstallationLanguage.jsx";
 import { Summary } from "./Summary.jsx";
-import { AddressContext } from "./Common.jsx";
+import { AddressContext, ConfContext } from "./Common.jsx";
+import { readConf } from "./helpers/conf.js";
 
 import { usePageLocation } from "hooks";
 
 export const Application = () => {
     const [address, setAddress] = useState();
+    const [conf, setConf] = useState();
     const { path } = usePageLocation();
 
     useEffect(() => cockpit.file("/run/anaconda/bus.address").watch(setAddress), []);
+    useEffect(() => readConf().then(data => {
```

Comment:
I know you dont fancy shortHands, but just for your interest this is very often writen like:

``` js
    useEffect(() => readConf().then(setConf, ex => ex => console.error("Failed to parse anaconda configuration")), []);
```